bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -511,8 +511,10 @@ public Void call() throws Exception {
                             ex.getMessage(), ex);
                 }
             } else {
-                logger.debug("Cannot notify handler about updated thing {}, because thing is not initialized "
-                        + "(must be in status ONLINE or OFFLINE).", thing.getThingTypeUID());
+                logger.debug(
+                        "Cannot notify handler about updated thing '{}', because thing is not initialized (must be in status ONLINE or OFFLINE). Start handler initialization instead.",
```

Comment:
I have been told some time ago that it is not the thing that is (or is not) initialized, it is the handler only.
It is currently derived by the thing status, but that should be already abstracted by the isHandlerInitialized function above.